packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.types.ts

```diff
@@ -122,6 +122,32 @@ export interface ICheckboxProps extends React.ButtonHTMLAttributes<HTMLElement |
    * Optional keytip for this checkbox
    */
   keytipProps?: IKeytipProps;
+
+  /**
+   * Optional controlled indeterminate visual state for checkbox. Use this if you want to control
+   * indeterminate state. Setting indeterminate state takes visual precedence
+   * over checked or defaultChecked props given but does not affect checked state.
+   * This is should not be a toggleable state. On load the checkbox will receive indeterminate visual state
+   * and after the first user click it should be removed by your supplied removeIndeterminate callback
```

Comment:
>click [](start = 30, length = 5)

clicks, not click

Reply:
Would it be altogether better to write "...user's first click..."?

Reply:
OH, I misinterpreted :) Sorry :$ 

Yeah I think user's first click would be better :)